src/Microsoft.Data.Sqlite/SqliteCommand.cs

```diff
@@ -88,19 +88,22 @@ public override void Prepare()
 
         public new virtual SqliteDataReader ExecuteReader(CommandBehavior behavior)
         {
-            if (behavior != CommandBehavior.Default)
+            if (behavior != CommandBehavior.Default && behavior != CommandBehavior.SequentialAccess)
```

Comment:
This is a flags enum. The logic here works by coincidence since `Default` is `0`, but there may be a clearer way to express it. Something like...
```C#
behavior & ~(CommandBehavior.Default | CommandBehavior.SequentialAccess) != 0
```
```